root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+      [% l('The {doc|annotation} field functions like a miniature wiki.',
+      { doc => { href => doc_link('Annotation'), target => '_blank' } }) %]
     </p>
     <p>
-      [% l('The purpose of this field is to add information that usually doesn\'t fit into the strict structural data schema of MusicBrainz (be it due to technical limitations that may be addressed later, or because the information in itself has to be free-text).') %]
+      [% l('The purpose of this field is to add information that usually
+            doesn’t fit into the strict structural data schema of MusicBrainz
+            (be it due to technical limitations that may be addressed later,
+            or because the information in itself has to be free-form text).') %]
     </p>
     <p>
-      [% l('This annotation will be published under an open license (<a href="{url}" target="_blank">CC BY-NC-SA 3.0</a>) and as such, it should not contain text that you don\'t have the right to release under that license. While you can quote a source to support a point you\'re making, you should never enter promotional texts or other artist or label-owned texts into the annotation.',
-      {url => "https://creativecommons.org/licenses/by-nc-sa/3.0/"}) %]
+      [% l('This annotation will be published under an open license
+            ({url|CC BY-NC-SA 3.0}) and as such, it should not contain text
+            that you don’t have the right to release under that license.
+            While you can quote a source to support a point you’re making,
+            you should never enter promotional text or other artist- or
+            label-owned text into the annotation.',
+           { url => { href => "https://creativecommons.org/licenses/by-nc-sa/3.0/", target => '_blank' } }) %]
     </p>
   </div>
 
   <div class="bubble" data-bind="bubble: $root.commentBubble">
     <p>
-      [% l('The disambiguation field is used to help users distinguish between identically named releases.') %]
+      [% l('The {doc|disambiguation} field is used to help users distinguish between identically-named releases.',
+           { doc => { href => doc_link('Disambiguation_Comment'), target => '_blank' } }) %]
     </p>
     <p>
-      [% l('This field is not a place to store general background information about the release: that kind of information should go in the annotation field.') %]
+      [% l('This field is not a place to store general background information about the release.
+            That kind of information should go in the annotation field.') %]
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.statusBubble">
+  <div class="bubble" data-bind="bubble: $root.externalLinkBubble">
+    <p>
+      [% l('External links are URLs associated with this particular release.
+            Mail-order links shouldn’t be attached to digital media releases,
+            and download links shouldn’t be attached to CD or vinyl releases.') %]
```

Comment:
I figured it was worth mentioning this here, as e.g. including mail-order Bandcamp links on digital media releases is a problem that I see frequently.

Reply:
I agree with indicating this. Would it make sense to separate it in two paragraphs? Just "External links are URLs associated with this particular release." on the first, then the purchase links on the second? Also, you should say "download and streaming links" probably.

Reply:
Makes sense! I've made these changes locally.